cmd/argocd/commands/app.go

```diff
@@ -957,6 +957,11 @@ func NewApplicationWaitCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 		watchSuspended  bool
 		watchOperations bool
 		timeout         uint
+		resources       *[]string
+	)
+	const (
+		resourceFieldDelimiter = ":"
+		resourceFieldCount     = 3
```

Comment:
I think we can move these consts outside the function now since they are used in two places now